charts/index.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v1
 entries:
   alb-ingress:
   - appVersion: 1.0-beta.7
-    created: 2019-07-19T17:47:00.627292+02:00
+    created: "2019-10-14T17:41:26.298267+01:00"
     description: The ALB Ingress Controller satisfies Kubernetes ingress resources
       by provisioning Application Load Balancers.
-    digest: 9aa9bf52878c90b29630f356266f8725735be02ad7e149d9a3d33936b6ba8622
+    digest: ec376ef8abde3bd462ce924d712fe78263b26ebc63ad5eff23d8011913f6618c
```

Comment:
Why are there changes in the digest in some places and not others. Will chanting the digest break helm deployments that already use it?

Reply:
I don't think the package command was ran after some charts were updated & caused the changes in some but not all. I'm not sure whether it'll break anything - I'll have to investigate further for it